src/containers/PermissionsApplication.jsx

```diff
@@ -84,16 +102,42 @@ const PermissionsApplication = ({ match, t }) => {
         </Typography>
       ) : (
         <div>
-          <PageTitle>{appName.toUpperCase()}</PageTitle>
-          {sortPermissionsByName(matchingQueryResult).map(
-            ({ name, title, verbs }) => (
-              <Link to={`/permissions/${appName}/${name}`} key={name}>
-                <Typography variant="h4">
-                  {title} : {t(displayPermissions(verbs))}
-                </Typography>
-              </Link>
-            )
-          )}
+          <NavigationList>
+            <PageTitle>{appName.toUpperCase()}</PageTitle>
+            <NavigationListSection>
+              {sortPermissionsByName(matchingQueryResult).map(
+                ({ name, title, verbs, type }) => {
+                  const iconName = getPermissionIconName(type)
+                  return (
+                    <div key={name}>
+                      <Link
+                        to={`/permissions/${appName}/${name}`}
+                        key={name}
+                        style={{ textDecoration: 'none' }}
+                      >
+                        <ListItem button>
+                          <ListItemIcon>
+                            <Icon
+                              icon={
+                                require(`cozy-ui/transpiled/react/Icons/${iconName}`)
```

Comment:
This line adds 70kB in the vendors/settings.[hash].js . I wonder if there would be a better implementation @Crash--